New package: TensorCrossInterpolation v0.8.1

JuliaRegistrator opened the pull request:
- Registering package: TensorCrossInterpolation
- Repository: https://gitlab.com/tensors4fields/tensorcrossinterpolation.jl
- Created by: https://gitlab.com/terasakisatoshi
- Version: v0.8.1
- Commit: 72f84f90a8e2cb650d5de12c9ccae522a97a5965
- Git reference: v0.8.1
- Description: Online manual: https://tensors4fields.gitlab.io/tensorcrossinterpolation.jl/dev/index.html

terasakisatoshi commented:
[noblock]

❓ 

> Repo URL does not end with /name.jl.git, where name is the package name

terasakisatoshi commented:
[noblock]

❓ 

> Repo URL does not end with /name.jl.git, where name is the package name

JuliaHubRegistrator committed:
New package: TensorCrossInterpolation v0.8.1

UUID: b261b2ec-6378-4871-b32e-9173bb050604
Repo: https://gitlab.com/tensors4fields/tensorcrossinterpolation.jl.git
Tree: 51e1c4d8824e2abf501ac2aacd7e967cc0ea4687

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

terasakisatoshi commented:
I have submitted [another pull request](https://github.com/JuliaRegistries/General/pull/102108) for registry registration with the URL in PascalCase. Therefore, please feel free to close this pull request.
